src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnectionOptions.cs

```diff
@@ -262,6 +262,8 @@ public CookieContainer Cookies
     /// <remarks>
     /// This delegate is invoked after headers from <see cref="Headers"/> and the access token from <see cref="AccessTokenProvider"/>
     /// has been applied.
+    /// <para />
+    /// If <c>ClientWebSocketOptions.HttpVersion</c> is set to <c>2.0</c> or higher, some options like <see cref="ClientWebSocketOptions.Cookies"/> will not be applied. Instead use <see cref="Cookies"/> or the corresponding option on <see cref="HttpConnectionOptions"/>.
```

Comment:
Are we are setting it to 2.0 before calling this Action? If 2.0 is not the normal default and we're changing it, we should probably mention this in the doc comment.

Reply:
Nope, it's opt-in, especially because of the behavior change with cookies/certs/etc.